src/java.base/share/man/java.1

```diff
@@ -1460,6 +1460,15 @@ usage by individual \f[V]CallSite\f[R], individual virtual memory region
 and its committed regions.
 .RE
 .TP
+\f[V]-XX:TrimNativeHeapInterval\f[R]
+Interval, in ms, at which the JVM will trim the native heap. Lower values
+will reclaim memory more eagerly at the cost of higher overhead. A value
+of 0 (default) disables native heap trimming.
```

Comment:
Should we add:
```
Native heap trimming is performed in a dedicated thread.
```
so people understand the potential cost of enabling this a little better?